app/javascript/scavenger_hunt/scenes/Questions/QuestionThree/QuestionThree.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { FormHelperText } from '@material-ui/core';
+import { Card, Button } from "react-bootstrap";
+import DatePicker from "react-datepicker";
+import './QuestionThree.scss';
+import "react-datepicker/dist/react-datepicker.css";
+
+const  QuestionThree = () => {
+  const [DateOne, setDateOne] = useState(new Date());
+  const [DateTwo, setDateTwo] = useState(new Date());
+
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log("Answer Submitted is: ", DateOne, DateTwo);
```

Comment:
<img width="289" alt="Screen Shot 2020-09-02 at 8 20 07 PM" src="https://user-images.githubusercontent.com/46115782/92049952-c6b75f80-ed59-11ea-83af-cf1ad54b4fee.png">

Up to you to convert the date to ```mm/dd/yyyy``` when validating the answers

Reply:
should we use that format instead of `dd/mm/yyyy`?

Reply:
Done